src/test/java/org/jenkinsci/plugins/gitclient/CliGitCommand.java

```diff
@@ -58,9 +59,9 @@ private String[] run(boolean assertProcessStatus) throws IOException, Interrupte
         ByteArrayOutputStream bytesErr = new ByteArrayOutputStream();
         Launcher.ProcStarter p = launcher.launch().cmds(args).envs(env).stdout(bytesOut).stderr(bytesErr).pwd(dir);
         int status = p.start().joinWithTimeout(1, TimeUnit.MINUTES, listener);
-        String result = bytesOut.toString("UTF-8");
+        String result = bytesOut.toString(StandardCharsets.UTF_8);
         if (bytesErr.size() > 0) {
-            result = result + "\nstderr not empty:\n" + bytesErr.toString("UTF-8");
+            result = result + "\nstderr not empty:\n" + bytesErr.toString(StandardCharsets.UTF_8);
```

Comment:
Using newer Java Platform functionality that avoids the messiness around `UnsupportedEncodingException`.